system/lib/wasmfs/syscalls.cpp

```diff
@@ -1389,6 +1391,82 @@ int __syscall_fcntl64(int fd, int cmd, ...) {
   }
 }
 
+intptr_t _mmap_js(intptr_t addr,
+                  size_t length,
+                  int prot,
+                  int flags,
+                  int fd,
+                  size_t offset,
+                  int* allocated) {
+  if (addr != 0) {
+    // We don't currently support location hints for the address of the mapping.
+    return -EINVAL;
```

Comment:
Since it's a hint, would it be better to just ignore it?

Reply:
Good idea... reading the docs, Linux at least in some cases may ignore the hint. So we can too. However, meanwhile @sbc100 refactored away that argument entirely :smile: